examples/vae_text/vae_train.py

```diff
@@ -243,23 +250,11 @@ def decode(self,
 class VAEData(tx.data.MonoTextData):
     r""" Wrap `tx.data.MonoTextData` to implement data preprocessing.
     """
-    def process(self, raw_example: str) -> List[str]:
+    def process(self, raw_example: List[str]) -> List[str]:
         # Truncates sentences and appends BOS/EOS tokens.
-        words = raw_example.strip().split(self._delimiter)
-        if (self._max_seq_length is not None and
-                len(words) > self._max_seq_length):
-            if self._length_filter_mode == "truncate":
-                words = words[:self._max_seq_length]
-
-        if self._hparams.dataset["bos_token"] != '':
-            words.insert(0, self._hparams.dataset["bos_token"])
-        if self._hparams.dataset["eos_token"] != '':
-            words.append(self._hparams.dataset["eos_token"])
-
-        # Apply the "other transformations".
-        for transform in self._other_transforms:
-            words = transform(words)
-
+        raw_example = super().process(raw_example)
+        words = " ".join(raw_example[1:-1]).strip().split(self._delimiter)
+        words = [raw_example[0]] + words + [raw_example[-1]]
```

Comment:
I don't understand what you're doing here. How will `words` differ from `raw_example`?

Reply:
Also, #118 is already fixed.